<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeParentView</class>
 <widget class="QWidget" name="TreeParentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OctoVault VPN</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lblParentImage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/img/images/icon_eti_gray.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblParentName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>13</y>
     <width>110</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>Country Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblParentArrow">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>15</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets/assets.qrc">:/img/images/icon_front_arrow_gray.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblParentSignal">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>15</y>
     <width>15</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets/assets.qrc">:/img/images/icon_network_high.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
